usaspending_api/submissions/management/commands/load_submission.py

```diff
@@ -120,18 +113,9 @@ def handle(self, *args, **options):
         # Create account objects
         for row in appropriation_data:
             # Check and see if there is an entry for this TAS
-            treasury_account = TreasuryAppropriationAccount.objects.filter(tas_rendering_label=row['tas']).first()
+            treasury_account = get_treasury_appropriation_account_tas_lookup(row['tas_id'], db_cursor)
             if treasury_account is None:
-                treasury_account = TreasuryAppropriationAccount()
-
-                field_map = {
-                    'tas_rendering_label': 'tas',
-                    'submission_id': submission_attributes.submission_id,
-                    'allocation_transfer_agency__cgac_code': 'allocation_transfer_agency',
-                    'responsible_agency__cgac_code': 'agency_identifier'
-                }
-
-                load_data_into_model(treasury_account, row, field_map=field_map, save=True)
+                raise Exception('Could not find appropriation account for TAS: ' + row['tas'])
```

Comment:
Thanks for the clarification--added [DS-536](https://federal-spending-transparency.atlassian.net/browse/DS-536) in case we want to think about this more when we start loading multiple submissions at a time.